### DISCUSSION TITLE
✅Embedded Web Browser

### DISCUSSION THREAD

**Discussion by jarrah31** [isontheline/pro.webssh.net#1145] [General]
Hi Arnaud, is it possible to access the embedded web browser on an iPad? If so how do I find it please?
The documentation says “Web Browser is NOT available on eWebSSH and WebSSH::o versions”. What does webssh::o mean?

**Comment by isontheline** [comment]
Hi @jarrah31 👋

Yes the embedded web browser is available on iPad but at least iOS 17 (needs features inside the latest firmware only).

eWebSSH and WebSSH::o are now deprecated and unavailable, were only available through China Mainland.

**Reply by jarrah31** [reply]
That’s great, thanks. I’m using the latest iPadOS 17.x but can’t see how I access the embedded web browser?
I have set up a “Forwardings” connection using Dynamic Port Forwarding and established a connection, but can’t find any button or menu option to access the embedded web browser. My version is 26.6.1365.

**Reply by isontheline** [reply]
Two options to launch WebBrowser :

1. From tools :
<img width="362" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/00c0653d-ce0b-47e4-ab87-b882e9251b21">

2. From DPF context menu :
<img width="386" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/be23f37c-5944-4683-8158-ca9bbae253f5">

**Reply by jarrah31** [reply]
Ahhh thank you, obvious when you know how! 😁
To help dummies like me, would you mind including where to find web browser on this help page please?
https://webssh.net/documentation/web-browser/

**Reply by isontheline** [reply]
Thank you @jarrah31 for your feedback 🙏

Just updated the documentation : https://webssh.net/documentation/web-browser/

**Comment by jarrah31** [comment]
Thank you Arnaud!
